src/znc.cpp

```diff
@@ -628,6 +629,7 @@ bool CZNC::WriteNewConfig(const CString& sConfigFile) {
     VCString vsLines;
 
     vsLines.push_back(MakeConfigHeader());
+    vsLines.push_back("PidFile = /run/znc/znc.pid");
```

Comment:
Well, remember: This only sets the default value. In your config you would change that.

However that's why I added the comment in the PR that I can provide a mechanism to specify default pidfile via configure.